extensions/amp-base-carousel/0.1/test/test-carousel.js

```diff
@@ -298,23 +308,23 @@ describes.realWin('carousel implementation', {}, (env) => {
         slideCount: 3,
         initialIndex: -1,
       });
-      expect(carousel.isAtStart()).to.be.true;
+      expect(carousel.scrollContainer_.scrollLeft).to.equal(0);
```

Comment:
Is this still needed if we modify `isAtStart()` instead?

Reply:
I made this change for the same reason above, since I wanted to keep the test inline with what `isAtStart` testing. But I can add both in for completenesses sake.